code/modules/food_and_drinks/machinery/gibber.dm

```diff
@@ -15,6 +15,12 @@
 
 /obj/machinery/gibber/Initialize(mapload)
 	. = ..()
+	if(prob(5))
+		name = "meat grinder"
+		desc = "Okay, if I... if I chop you up in a meat grinder, and the only thing that comes out, that's left of you, is your eyeball, \
+	you'r- you're PROBABLY DEAD! You're probably going to - not you, I'm just sayin', like, if you- if somebody were to, like, \
```

Comment:
im gonna cry

Reply:
indent ONCE

starting from desc

indent ONCE

Reply:
dont worry im crying too
doesnt this leave it literally how it was in the initial commit

Reply:
oh wow nvm initial commit was two indents from desc ok im dumb